controllers/v1/reports.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * name : reports.js
+ * author : Rakesh
+ * created-date : 01-Nov-2020
+ * Description : Reports related information.
+ */
+
+// Dependencies
+const csv = require('csvtojson');
```

Comment:
@rakeshSgr - Few generic comments.

Remove csvtojson module if not required.
Make all the APIs in this group a get call with query parameters
Correct spelling mistakes
Use let instead of var
Indent all the APIdoc properly

Cc - @aman-tunerlabs